site/src/components/indicator/forceIndex.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: component
+title: Force Index
+component: indicator/algorithm/forceIndex.js
+tags:
+namespace: indicator
+
+example-code: |
+
+  // Create and apply the Force Index algorithm
+  var forceAlgorithm = fc.indicator.algorithm.forceIndex();
+  forceAlgorithm(data);
+  
+  // Calculate the maximum absolute value of the index
+  var aboluteMax = Math.max(Math.abs(d3.min(data, function(d) { return d.force; })),
+    d3.max(data, function(d) { return d.force; }));
+  
+  //Scailing the display using the maximum absolute value of the Index
+  var yScale = d3.scale.linear()
+          .domain([-aboluteMax, aboluteMax])
+          .range([height - 10, 10]);
```

Comment:
Would `.nice()` give padding rather than using an explicit value? (It took me a while to work out what the 10 was for.